models/wishlist.model.js

```diff
@@ -0,0 +1,73 @@
+const connection = require('../connections/mysql_db');
+
+const getWishlist = (userId) => {
+    let query = {
+        sql: `select * from wishlist as w
```

Comment:
in pgSql we could do something like property.* I don't know if this works in mysql too could you try

Reply:
yes it does work

Reply:
can you change to that , I accidentally merged you pr sorry